src/yaml/wallibuk/30944-south-american-houses-pack.yaml

```diff
@@ -0,0 +1,47 @@
+group: wallibuk
+name: south-american-houses-pack
+version: "2.0.0"
+subfolder: "200-residential"
+info:
+  summary: South American houses PACK
+  description: |-
+    **I dont recommend mixing of both housing packs together (brazil and peru)- use separately!!!!**
+
+    Size of BATs: only 1x2
+
+    Growth stage: 1st, 2nd
+
+    Dependencies: [BSC Mega Props - CP Vol02](https://www.sc4devotion.com/csxlex/lex_filedesc.php?lotGET=2790) (recommend)
+  author: Wallibuk
+  website: https://community.simtropolis.com/files/file/30944-south-american-houses-pack/
+  images:
+    - https://www.simtropolis.com/objects/screens/monthly_2018_08/82.jpg.9a80bfb5906977ef57267a2a0670d08b.jpg
+    - https://www.simtropolis.com/objects/screens/monthly_2018_08/85.jpg.3e3cb3478085e7b0b46c000c851ef2fc.jpg
+    - https://www.simtropolis.com/objects/screens/monthly_2018_08/84.jpg.39f301fed0e5cb9ef50ff3834deaa12f.jpg
+    - https://www.simtropolis.com/objects/screens/monthly_2018_08/80.jpg.dc04e10e93e1b6fa61745b792ff606f0.jpg
+    - https://www.simtropolis.com/objects/screens/monthly_2018_08/81.jpg.e90514d88572a71764955032f9eb5893.jpg
+    - https://www.simtropolis.com/objects/screens/monthly_2018_08/79.jpg.bdd786fe2fa7afee937c9cefea56349c.jpg
+dependencies:
+  - bsc:mega-props-cp-vol02
+assets:
+  - assetId: wallibuk-south-american-houses-pack-part0
```

Comment:
I'd rename them to `-brazil`, `-peru` & `chapels-churches`. The `part` suffix is used if there are multiple assets and the script can't figured out what they're used for (like maxisnite/darknite), so I tend to name them manually.